### PR TITLE
Two new power skills added: Custom NER and Custom Text Classification

### PR DESCRIPTION
**Pull request by ignaciofls**
## Purpose
Two new power skills added: Custom NER and Custom Text Classification

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: Text/CustomNER/custom_ner/__init__.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+# Function to submit the analysis job towards the Text Analytics (TA) API
+def get_entities (value):
+    language = str(value['data']['lang'])
+    corpus = str(value['data']['text'])
+    #To be filled with your Text Analytics details: endpoint, key, deployment name and project name inside your appsettings (https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings)
+    endpoint = os.environ["TA_ENDPOINT"] # This will look like 'https://westeurope.api.cognitive.microsoft.com/text/analytics/v3.2-preview.2/analyze' to be updated once product goes GA
+    key = os.environ["TA_KEY"]
+    project_name = os.environ["PROJECT_NAME"]
+    deployment =  os.environ["DEPLOYMENT"]
+    body = "{'displayName': 'Extracting custom NERS', 'analysisInput': {'documents': [{'id': '1', 'language': '" + language + "', 'text':'" + corpus + "'}]}, 'tasks': {'customEntityRecognitionTasks': [{'parameters': {'project-name': '"+ project_name +"','deployment-name': '"+ deployment +"'}}]}}"
```
Comment (bleroy) [inline Text/CustomNER/custom_ner/__init__.py:84]: This looks dangerous. Shouldn't we be encoding those user inputs before we concatenate?

Comment (bleroy) [reply Text/CustomNER/custom_ner/__init__.py:84]: Or better, build a real object, then serialize that.

File: Text/CustomTextClassifier/README.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+          "sortable": false,
+          "analyzer": null,
+          "indexAnalyzer": null,
+          "searchAnalyzer": null,
+          "synonymMaps": [],
+          "fields": []
+        }
+      ]
+    }
+```
```
Comment (bleroy) [inline Text/CustomTextClassifier/README.md:184]: Missing a closing brace.

File: Text/CustomTextClassifier/customtextcla/__init__.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    # HTTPConnection.debuglevel = 1 
+
+    language = str(value['data']['lang'])
+    corpus = str(value['data']['text'])
+    #To be filled with your Text Analytics details: endpoint, key, deployment name and project name inside your appsettings (https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings)
+    endpoint = os.environ["TA_ENDPOINT"] # This will look like 'https://westeurope.api.cognitive.microsoft.com/text/analytics/v3.2-preview.2/analyze' to be updated once product goes GA
+    key = os.environ["TA_KEY"]
+    project_name = os.environ["PROJECT_NAME"]
+    deployment =  os.environ["DEPLOYMENT"]
+    body = "{'displayName': 'Extracting custom text classification', 'analysisInput': {'documents': [{'id': '1', 'language': '" + language + "', 'text':'" + corpus + "'}]}, 'tasks': {'customMultiClassificationTasks': [{'parameters': {'project-name': '"+ project_name +"','deployment-name': '"+ deployment +"'}}]}}"
```
Comment (bleroy) [inline Text/CustomTextClassifier/customtextcla/__init__.py:94]: same as for the other skill: build an object, then serialize.

**Review by bleroy — CHANGES_REQUESTED** [review]
Cool stuff, thanks for the contribution. I have some minor comments that should not be too time-consuming to address, hopefully.

File: README.md [hunk 4]
```diff
@@ -18,6 +18,7 @@ This project provides the following custom skills:
...
 | [BingEntitySearch](Text/BingEntitySearch/README.md) | finds rich and structured information about public figures, locations, or organizations. | Text | ![C#](https://img.shields.io/badge/language-C%23-brightgreen) | ![functions](https://img.shields.io/badge/deploy-Functions-blue) | ARM Template |
 | [CustomEntityLookup](/Text/CustomEntitySearch) | finds custom entity names in text. A custom skill implementation of the [custom entity lookup skill](https://docs.microsoft.com/en-us/azure/search/cognitive-search-skill-custom-entity-lookup), consider using in the cognitive skill instead of this custom skill implementation.   | Text|![C#](https://img.shields.io/badge/language-C%23-brightgreen) | ![functions](https://img.shields.io/badge/deploy-Functions-blue) | ARM Template |
+| [CustomNER](/Text/CustomNER) | extracts your custom entities, using NLP with [Text Analytics Custom NER](https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/custom-named-entity-recognition/overview)  | Text|![python](https://img.shields.io/badge/language-python-orange) | ![functions](https://img.shields.io/badge/deploy-Functions-blue) | Manual |
```
[the remark this answers is not in the document]
Comment (ignaciofls) [reply README.md:21]: solved, will send a new PR with all the suggested changes. thanks

File: Text/CustomNER/README.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
+# custom_ner_skill
+---
+Description:
+- It is common to have custom entities along different texts that dont fit any of the predefined entities that can be extracted with Named Entity Extraction service. Custom Named Entity Recognition (in preview as of Nov2021) provides the capability to ingest your training texts, label your set of custom entities and train a model to identify them. You can easily deploy the model in a secured fashion to later on run your inference along your texts. As an outcome you will get the detected custom entities, their position (inside the text) and the confidence level
```
[the remark this answers is not in the document]
Comment (ignaciofls) [reply Text/CustomNER/README.md:4]: solved, will send a new PR with all the suggested changes. thanks

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ignaciofls) [reply README.md:21]: solved, will send a new PR with all the suggested changes. thanks

File: Text/CustomNER/custom_ner/__init__.py (hunk 1 above)
Comment (ignaciofls) [reply Text/CustomNER/custom_ner/__init__.py:84]: solved, will send a new PR with all the suggested changes. thanks

**Comment by ignaciofls** [comment]
Closing this one, just fixed the changes. will create a new PR

**Pull request closed without merging**